docs/docs/development/documentation.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Documentation
+sidebar_label: Documentation
+---
+
+:::info Please Note
+This page outlines how to update the zmk documentation, not the zmk firmware sources.
+:::
+
+:::note
+The documentation is written with [Docusaurus](https://docusaurus.io/). The zmk source code has all of the necessary Docusaurus dependencies included but referencing their documentation can be helpful at times.
+:::
+
+:::note
+If you are working with the documentation from within VS Code+Docker please be aware the documentation will not be auto-generated when making changes while the server is running. You'll need to restart the server when saving changes to the documentation.
+:::
+
+## Preparation
+
+In order to view and verify documentation updates locally you'll need to install `Node.js` and `npm`. Please see [nodejs.org](https://nodejs.org/) for details on how to install these tools.
+
+## Verifying Documentation Updates Locally
+
+To verify documentation updates locally, follow the following procedure. The `npm` commands and first step will need to be run from a terminal.
+
+1. Navigate to the `docs` folder
+2. Run `npm ci` to build the necessary tools if you haven't run it before or the zmk sources were updated
+3. Run `npm start` to start the local server that will let you see your documentation updates via a web browser
+4. If a web browser doesn't open automatically: you'll need to open a browser window or tab and navigate to `http://localhost:3000` to view your changes
+5. Verify the changes look good
+6. Run the formatting and checks from the section below
+7. Submit any changes as a Pull Request
+
+## Formatting and Verifying Documentation Updates
+
+Prior to submitting a documentation pull request, you'll want to run the format and check commands. These commands are run as part of the verification process on pull requests so it's good to run them ahead of submitting documentation updates.
+
+The format command can be run with the following procedure in a terminal that's inside the zmk source directory.
+
+1. Navigate to the `docs` folder
+2. Run `npm run prettier:format`
+
+The check commands can be run with the following procedure in a terminal that's inside the zmk source directory.
+
+1. Navigate to the `docs` folder
+2. Run `npm run prettier:check`
+3. Run `npm run lint`
+4. Run `npm run build`
```

Comment:
Might be helpful to explicitly say that if any of these steps print an error, you should fix the error and run the checks again.